test/e2e/common/security_context.go

```diff
@@ -92,6 +95,69 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		})
 	})
 
+	Context("When creating a container with runAsNonRoot", func() {
+		rootImage := imageutils.GetE2EImage(imageutils.BusyBox)
```

Comment:
Just confirming that we still want to use `BusyBox` instead of `debian` here?

Also, is there a doc/issue I can read around the switch from `BusyBox` to `debian` base images? Just for my own curiosity :)